src/main/resources/swagger/rawls.yaml

```diff
@@ -3232,6 +3231,19 @@ definitions:
         type: string
         description: The id of this workflow
 
+  WorkflowQueueStatusResponse:
+    description: Information about the status of workflows and the workflow queue
+    required:
+      - estimatedQueueTimeMS
+      - workflowCountsByStatus
+    properties:
+      estimatedQueueTimeMS:
```

Comment:
Have we settled on an algorithm yet? I think the name and description of this field should reflect what the algorithm is actually measuring. If it is the time that the last submitted workflow waited then maybe lastWorkflowPendTimeMS